apps/pollbook/frontend/src/unconfigured_screen.tsx

```diff
@@ -124,6 +123,20 @@ export function UnconfiguredSystemAdminScreen(): JSX.Element {
     (p) => p.electionId && p.status === PollbookConnectionStatus.WrongElection
   );
   if (!isOnline || configuredPollbooks.length <= 0) {
+    if (hadConfigurationError) {
+      return (
+        <FullScreenMessage
+          title="Failed to configure VxPollBook"
+          image={
+            <FullScreenIconWrapper>
+              <Icons.Warning color="warning" />
+            </FullScreenIconWrapper>
+          }
+        >
+          Error configuring machine please try again.
```

Comment:
nit: can we make this two sentences? "Error configuring machine. Please try again."